addons/mass_mailing/models/mass_mailing.py

```diff
@@ -362,7 +362,7 @@ def _get_mailing_model(self, cr, uid, context=None):
             ondelete='set null',
         ),
         'state': fields.selection(
-            [('draft', 'Draft'), ('test', 'Tested'), ('done', 'Sent')],
+            [('draft', _('Draft')), ('test', _('Tested')), ('done', _('Sent'))],
```

Comment:
What @antonylesuisse meant is that this particular change is incorrect, because selection field labels are always automatically translated and must not be surrounded by manual calls to the translation function (`_()`)